arches/app/models/fields/i18n.py

```diff
@@ -248,6 +249,8 @@ def _parse(self, value, lang, use_nulls):
             ret = value.raw_value
         elif isinstance(value, dict):
             ret = value
+        elif isinstance(value, BaseExpression):
```

Comment:
Yeah, it's worth pulling out now. This was an artifact of trying to salvage the only part of the pre-controlled-list-manager RDM prototyping I did on the domain datatype.

Also, since that time, this was [reported as a bug](https://code.djangoproject.com/ticket/35167) against Django 4.2, which suggests a workaround I can use instead. Will pull out into a new PR. 👍